zeebe/broker/src/main/java/io/camunda/zeebe/broker/partitioning/startup/RaftPartitionFactory.java

```diff
@@ -24,10 +24,12 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.time.Duration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public final class RaftPartitionFactory {
   public static final String GROUP_NAME = "raft-partition";
-
+  private static final Logger LOG = LoggerFactory.getLogger(RaftPartitionFactory.class);
```

Comment:
:wrench: May be we can use `io.camunda.zeebe.broker.Loggers.SYSTEM_LOGGER`

Reply:
Good catch 🙌